MF-282 - Rename clients to things in Dashflux

dborovcanin opened the pull request:
This pull request resolves #282.

dborovcanin committed:
Rename services, stores, environments and models

Rename services, stores and models to match naming change from `clients` to `things`.

Signed-off-by: Dušan Borovčanin <dusan.borovcanin@mainflux.com>

dborovcanin committed:
Replace all occurrences of clients with things

Signed-off-by: Dušan Borovčanin <dusan.borovcanin@mainflux.com>

nmarcetic commented on dashflux/src/app/app.component.spec.ts:
```diff
@@ -7,7 +7,7 @@ import { MaterialModule } from './core/material/material.module';
 import { TokenStorage } from './core/services/auth/token-storage.service';
 import { ChannelsService } from './core/services/channels/channels.service';
-import { ClientsService } from './core/services/clients/clients.service';
+import { ThingsService } from './core/services/clients/clients.service';
```
Rename to things service.

nmarcetic commented on dashflux/src/app/app.module.ts:
```diff
@@ -16,19 +16,19 @@ import { LoginComponent } from './components/auth/login/login.component';
 import { TokenStorage } from './core/services/auth/token-storage.service';
 import { ChannelsService } from './core/services/channels/channels.service';
-import { ClientsService } from './core/services/clients/clients.service';
+import { ThingsService } from './core/services/clients/clients.service';
```
Rename.

nmarcetic commented on dashflux/src/app/app.module.ts:
```diff
@@ -16,19 +16,19 @@ import { LoginComponent } from './components/auth/login/login.component';
+import { ThingsService } from './core/services/clients/clients.service';
 import { MockAuthService } from './core/services/mock-auth.service';
 import { MockChannelsService } from './core/services/mock-channels.service';
 import { MockClientsService } from './core/services/mock-clients.service';
```
Rename.

nmarcetic commented on dashflux/src/app/app.module.ts:
```diff
@@ -61,13 +61,13 @@ export function factory(authenticationService: AuthenticationService) {
     ChannelsStore,
     AuthStore,
     MockAuthService,
     MockClientsService,
```
Rename.

nmarcetic commented on dashflux/src/app/components/channels/add-channel-dialog/add-channel-dialog.component.html:
```diff
@@ -11,12 +11,12 @@ <h1 *ngIf="editMode" mat-dialog-title>Edit channel</h1>
       </mat-form-field>
       <mat-form-field>
           <mat-select placeholder="Connected clients" formControlName="connected" [compareWith]="compareFunction" multiple>
```
rename.

nmarcetic commented on dashflux/src/app/components/channels/add-channel-dialog/add-channel-dialog.component.html:
```diff
@@ -11,12 +11,12 @@ <h1 *ngIf="editMode" mat-dialog-title>Edit channel</h1>
       <mat-form-field>
           <mat-select placeholder="Connected clients" formControlName="connected" [compareWith]="compareFunction" multiple>
-            <mat-option *ngFor="let client of clientsStore.clients" [value]="client">{{client.name}}</mat-option>
+            <mat-option *ngFor="let client of clientsStore.things" [value]="client">{{client.name}}</mat-option>
```
Rename.

nmarcetic commented on dashflux/src/app/components/channels/add-channel-dialog/add-channel-dialog.component.spec.ts:
```diff
@@ -9,9 +9,9 @@ import { MaterialModule } from '../../../core/material/material.module';
 import { TokenStorage } from '../../../core/services/auth/token-storage.service';
 import { ChannelsService } from '../../../core/services/channels/channels.service';
-import { ClientsService } from '../../../core/services/clients/clients.service';
+import { ThingsService } from '../../../core/services/clients/clients.service';
```
Rename.

nmarcetic commented on dashflux/src/app/components/channels/add-channel-dialog/add-channel-dialog.component.ts:
```diff
@@ -20,7 +20,7 @@ export class AddChannelDialogComponent implements OnInit {
     private dialogRef: MatDialogRef<AddChannelDialogComponent>,
     @Inject(MAT_DIALOG_DATA) public data: Channel,
-    public clientsStore: ClientsStore,
+    public clientsStore: ThingsStore,
```
Rename.

nmarcetic commented on dashflux/src/app/components/channels/channels.component.spec.ts:
```diff
@@ -9,9 +9,9 @@ import { MaterialModule } from '../../core/material/material.module';
 import { TokenStorage } from '../../core/services/auth/token-storage.service';
 import { ChannelsService } from '../../core/services/channels/channels.service';
-import { ClientsService } from '../../core/services/clients/clients.service';
+import { ThingsService } from '../../core/services/clients/clients.service';
```
Rename.

nmarcetic commented on dashflux/src/app/components/channels/channels.component.ts:
```diff
@@ -19,13 +19,13 @@ export class ChannelsComponent implements OnInit {
   constructor(
     private dialog: MatDialog,
-    public clientsStore: ClientsStore,
+    public clientsStore: ThingsStore,
```
Rename.

nmarcetic commented on dashflux/src/app/components/channels/channels.component.ts:
```diff
@@ -19,13 +19,13 @@ export class ChannelsComponent implements OnInit {
   ngOnInit() {
     this.channelsStore.getChannels();
-    this.clientsStore.getClients();
+    this.clientsStore.getThings();
```
Rename.

nmarcetic commented on dashflux/src/app/components/things/add-thing-dialog/add-thing-dialog.component.spec.ts:
```diff
@@ -9,16 +9,16 @@ import { MaterialModule } from '../../../core/material/material.module';
 import { ChannelsService } from '../../../core/services/channels/channels.service';
-import { ClientsService } from '../../../core/services/clients/clients.service';
-import { AddClientDialogComponent } from './add-client-dialog.component';
+import { ThingsService } from '../../../core/services/clients/clients.service';
```
Rename.

nmarcetic commented on dashflux/src/app/components/things/things.component.spec.ts:
```diff
@@ -9,19 +9,19 @@ import { MaterialModule } from '../../core/material/material.module';
 import { ChannelsService } from '../../core/services/channels/channels.service';
-import { ClientsService } from '../../core/services/clients/clients.service';
-import { ClientsComponent } from './clients.component';
+import { ThingsService } from '../../core/services/clients/clients.service';
```
Rename.

nmarcetic commented on dashflux/src/app/components/things/things.component.ts:
```diff
@@ -0,0 +1,63 @@
+  styleUrls: ['./things.component.scss']
+})
+export class ThingsComponent implements OnInit {
+  clients: Observable<Thing[]>;
```
Rename.

nmarcetic commented on dashflux/src/app/core/services/channels/channels.service.ts:
```diff
@@ -60,44 +60,44 @@ export class ChannelsService {
     });
   }
 
   getClientsToDelete(channelFormData: Channel, channel: Channel) {
```
Rename this and all above.

nmarcetic commented on dashflux/src/app/core/services/mock-clients.service.ts:
```diff
@@ -1,9 +1,9 @@
+import { Thing } from '../store/models';
 
-let MOCK_CLIENTS = [
+const MOCK_CLIENTS = [
```
Rename.

nmarcetic commented on dashflux/src/app/core/services/mock-clients.service.ts:
```diff
@@ -12,8 +12,8 @@ export class MockClientsService {
       return Observable.of(MOCK_CLIENTS).delay(1000);
```
Rename.

nmarcetic commented on dashflux/src/app/core/services/mock-clients.service.ts:
```diff
@@ -12,8 +12,8 @@ export class MockClientsService {
-  addClient(client: Client) {
-    MOCK_CLIENTS.push(client);
+  addClient(thing: Thing) {
+    MOCK_CLIENTS.push(thing);
```
Rename.

nmarcetic commented on dashflux/src/app/core/store/auth.store.spec.ts:
```diff
@@ -7,7 +7,7 @@ import { Observable } from 'rxjs/Observable';
 import { TokenStorage } from '../services/auth/token-storage.service';
 import { ChannelsService } from '../services/channels/channels.service';
-import { ClientsService } from '../services/clients/clients.service';
+import { ThingsService } from '../services/clients/clients.service';
```
Rename.

nmarcetic commented on dashflux/src/app/core/store/things.store.spec.ts:
```diff
@@ -0,0 +1,256 @@
+import { toJS } from 'mobx';
+import { Observable } from 'rxjs/Observable';
+
+import { ThingsService } from '../services/clients/clients.service';
```
Rename.

nmarcetic commented on dashflux/src/app/core/store/things.store.ts:
```diff
@@ -1,62 +1,62 @@
 
-import { ClientsService } from '../services/clients/clients.service';
-import { Client } from './models';
+import { ThingsService } from '../services/clients/clients.service';
```
Rename.

nmarcetic commented on dashflux/src/app/core/store/ui.store.spec.ts:
```diff
@@ -6,7 +6,7 @@ import { RouterTestingModule } from '@angular/router/testing';
 import { TokenStorage } from '../services/auth/token-storage.service';
 import { ChannelsService } from '../services/channels/channels.service';
-import { ClientsService } from '../services/clients/clients.service';
+import { ThingsService } from '../services/clients/clients.service';
```
Rename.

dborovcanin committed:
Update docker-compose and nginx.conf

Signed-off-by: Dušan Borovčanin <dusan.borovcanin@mainflux.com>

dborovcanin committed:
Rename `clients service` to `things service`

Signed-off-by: Dušan Borovčanin <dusan.borovcanin@mainflux.com>

dborovcanin committed:
Rename function and veriable names

Remove other occurences of `clients` word/prefix/sufix and replace it with `things`.

Signed-off-by: Dušan Borovčanin <dusan.borovcanin@mainflux.com>
